UnitTests/Schemas/IFC4X3_RC4/circular-arc_100.0_300_1000_1_Meter/src/circular-arc_100.0_300_1000_1_Meter.cpp

```diff
@@ -84,107 +84,15 @@ TEST_F(CircularArc, ImageIsSaved)
 TEST_F(CircularArc, PlaneSurfaceViews)
 {
 	// Arrange
-	const auto expected_front = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_front.png").string());
 	const auto expected_top = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_top.png").string());
-	const auto expected_bottom = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_bottom.png").string());
-	const auto expected_left = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_left.png").string());
-	const auto expected_right = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_right.png").string());
-	const auto expected_back = buw::loadImage4b(dataPath("circular-arc_100.0_300_1000_1_Meter_back.png").string());
-
-	// Act (Front)
-	rendererIfc->setViewDirection(buw::eViewDirection::Front);
-	buw::Image4b image_front = CaptureImage();
+
 	// Act (Top)
 	rendererIfc->setViewDirection(buw::eViewDirection::Top);
 	buw::Image4b image_top = CaptureImage();
-	// Act (Bottom)
-	rendererIfc->setViewDirection(buw::eViewDirection::Bottom);
-	buw::Image4b image_bottom = CaptureImage();
-	// Act (Left)
-	rendererIfc->setViewDirection(buw::eViewDirection::Left);
-	buw::Image4b image_left = CaptureImage();
-	// Act (Right)
-	rendererIfc->setViewDirection(buw::eViewDirection::Right);
-	buw::Image4b image_right = CaptureImage();
-	// Act (Back)
-	rendererIfc->setViewDirection(buw::eViewDirection::Back);
-	buw::Image4b image_back = CaptureImage();
 
 	// uncomment following lines to also save the screen shot
-	/*
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_front.png").string(), image_front);
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_top.png").string(), image_top);
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_bottom.png").string(), image_bottom);
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_left.png").string(), image_left);
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_right.png").string(), image_right);
-	buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_back.png").string(), image_back);
-	*/
+	//buw::storeImage(testPath("circular-arc_100.0_300_1000_1_Meter_top123.png").string(), image_top);
```

Comment:
A *...top123* image is committed as well. Is there a specific reason? Remove otherwise.